datalad/metadata/__init__.py

```diff
@@ -177,7 +177,7 @@ def get_metadata(ds, guess_type=False, ignore_subdatasets=False,
         for subds_path in ds.get_subdatasets(recursive=False):
             subds_meta_fname = opj(meta_path, subds_path, metadata_filename)
             if exists(subds_meta_fname):
-                subds_meta = json.load(open(subds_meta_fname, 'rb'))
+                subds_meta = json.loads(open(subds_meta_fname, 'rb').read().decode('utf-8'))
```

Comment:
fwiw I did just use 
```python
with open(filename) as f:
  j = json.load(f)
```
just fine in code elsewhere but I probably didn't have any encoded content.